### PR TITLE
US135051 - Quizzing UI - Adding mobile view for attempts dropdown

### PR DESCRIPTION
**Pull request by maddiegabriel**
[US135051 - [Consistent Eval][Quizzing] Make quizzing attempts dropdown responsive](https://rally1.rallydev.com/#/42960320374d/dashboard?detail=%2Fuserstory%2F621940993305)

In CE Assignments, we hide the file context dropdown for screens smaller than 930px to reduce clutter. This works because the user has the "Back to User Submissions" button to navigate between submissions.
image

However in CE Quizzing, the dropdown is currently the only way to navigate between attempts without leaving CE and coming back. Spoke with Gilberto about this, he asked for the dropdown should persist for smaller screen sizes in Quizzing 👍 

mobile_quiz_dropdown

### CODE REVIEW

File: components/header/consistent-evaluation-attempt-context.js [hunk 1]
```diff
@@ -42,8 +42,8 @@ export class ConsistentEvaluationAttemptContext extends RtlMixin(LocalizeConsist
 					white-space: nowrap;
 				}
 				@media (max-width: 930px) {
-					:host {
-						display: none;
+					.d2l-input-select {
+						max-width: 9rem;
```
Comment (jeroach) [inline components/header/consistent-evaluation-attempt-context.js:46]: How did we decide on this size for `max-width`?

Comment (maddiegabriel) [reply components/header/consistent-evaluation-attempt-context.js:46]: Good point, that size was leftover from hiding the dropdown on assignments. This code just shrinks the dropdown width so it fits better on smaller screens. I'll change it to `max-width: 767px` which is the mobile breakpoint we use elsewhere in CE

**Review by JamieTran — APPROVED** [review]
lgtm!

File: components/header/consistent-evaluation-attempt-context.js (hunk 1 above)
Comment (jeroach) [reply components/header/consistent-evaluation-attempt-context.js:46]: fyi I'm assuming that this update will be a part of this PR, and that's why I'm holding off on approving for now :)

let me know if that's not the case.

Comment (maddiegabriel) [reply components/header/consistent-evaluation-attempt-context.js:46]: oops I thought I pushed it up already, thanks for the reminder 😆

**Review by jeroach — APPROVED** [review]

**Pull request merged**